core_backend/app/utils.py

```diff
@@ -204,3 +204,11 @@ def get_http_client() -> aiohttp.ClientSession:
         http_client.start()
         _HTTP_CLIENT = http_client()
     return _HTTP_CLIENT
+
+
+def encode_api_limit(api_limit: int | None) -> int | str:
```

Comment:
/nit Is this used just once? We could just move this line to the function calling it. Maybe make it easier to understand

Reply:
It is used twice